### PR TITLE
Change to correct inhibit warnings use

### PR DESCRIPTION
**Pull request by MillerApps**
Docs: https://guides.cocoapods.org/syntax/podfile.html#inhibit_all_warnings_bang

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ target 'TrustKeystore' do
 
   pod 'BigInt'
   pod 'CryptoSwift', '~> 0.8.1'
-  pod 'TrezorCrypto', inhibit_warnings: true
-  pod 'TrustCore', inhibit_warnings: true
+  pod 'TrezorCrypto', :inhibit_warnings => true
+  pod 'TrustCore', :inhibit_warnings => true
```
Comment (alejandro-isaza) [inline Podfile:9]: Both of these are valid

**Review by alejandro-isaza — APPROVED** [review]

File: Podfile (hunk 1 above)
Comment (MillerApps) [reply Podfile:9]: Oh, I didn't know. Either way its probably better to keep consistent with the guide.

**Comment by vikmeup** [comment]
Do we need this?

**Comment by MillerApps** [comment]
We don't. Just wanted to keep consistent with the Cocoapods docs. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request closed without merging**